src/commands.rs

```diff
@@ -715,7 +715,7 @@ enum BranchSubcommand {
         names: Vec<String>,
 
         /// A glob pattern indicating branches to forget.
-        #[arg(action(ArgAction::Append), long = "glob")]
+        #[arg(long = "glob")]
```

Comment:
I think you might be able to remove the `="glob"` part and just have `arg(long)`.

Reply:
Good point. Done.